releases/24.0.0.1/full/Dockerfile.ubi.openjdk21

```diff
@@ -1,155 +1,48 @@
-FROM icr.io/appcafe/ibm-semeru-runtimes:open-21-jre-ubi9-minimal AS getRuntime
+ARG PARENT_IMAGE=icr.io/appcafe/open-liberty:24.0.0.1-kernel-slim-java21-openj9-ubi-minimal
 
-USER root
-
-ARG LIBERTY_VERSION=24.0.0.1
-ARG LIBERTY_SHA=c59edb53e8e88e674b67f0941e1abfc30eb0cd49
-ARG LIBERTY_DOWNLOAD_URL=https://repo1.maven.org/maven2/io/openliberty/openliberty-runtime/$LIBERTY_VERSION/openliberty-runtime-$LIBERTY_VERSION.zip
+FROM $PARENT_IMAGE AS installBundle
 
 ARG VERBOSE=false
+ARG LIBERTY_VERSION=24.0.0.1
+ARG FEATURES_SHA={replace_with_correct_sha}
```

Comment:
Need to fill in the sha for the features here?  Although I'm unsure what the default should be.  The one from the zip or the one from the json file.  I suspect the one from the json file should be the default since the build using the zip will always supply the sha as a build argument.

Reply:
It should the be the json file's SHA. Looked it up locally and specified it